enigma/src/main/java/org/quiltmc/enigma/api/translation/mapping/EntryMapping.java

```diff
@@ -34,6 +36,12 @@ public EntryMapping(@Nullable String targetName, @Nullable String javadoc) {
 		}
 	}
 
+	private static void validateSourcePluginId(String id) {
+		if (id != null && !id.matches("([a-z0-9_]+:[a-z0-9_]+)")) {
```

Comment:
Should we allow slashes in the path part?

Reply:
why?

Reply:
Mainly because our plugin has a single name proposal service and various "sub-services"

Reply:
sounds good!